en/docs/develop/customizations/adding-a-user-signup-workflow.md

```diff
@@ -182,6 +182,10 @@ This section explains how to attach a custom workflow to the user signup operati
 
 #### Engaging the WS Workflow Executor in the API Manager
 
+!!! important
+    When there are multiple tenants, first enable self sign up for tenant admin for each tenant by following the steps mentioned in [Enabling self signup]({{base_path}}/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup/#enabling-self-signup). 
+    Then to add a user sign-up workflow with regard to the Developer Portal, follow the below instructions from step 1 after sign in to the API-M Management Console as tenant admin for each tenant.
```

Comment:
Better skip short forms like 'API-M' in docs, check other places as well.

Reply:
Fixed with 10584cd96c089244ce8914132e6b047ff59d5453